DEHPEcosimPro/DstController/DstController.cs

```diff
@@ -706,7 +706,13 @@ public void GetNextExperimentStep()
             var cint = this.ReadNode(this.References
                 .FirstOrDefault(x => x.BrowseName.Name == "CINT")).Value;
 
-            var places = Convert.ToDouble(cint).ToString(CultureInfo.InvariantCulture).Split('.')[1];
+            var places = Convert.ToDouble(cint).ToString(CultureInfo.InvariantCulture);
+
+            // If we put 1 (for example) as Stepping, there is no '.' inside the string, invoking IndexOutOfRangeException
+            if (places.Contains('.'))
```

Comment:
Does the Invariant Culture always mean that the decimal seperator is a dot and can never be comma?

Reply:
As we convert the double to the string, I think it will always be a dot